app/org/sagebionetworks/bridge/services/email/MimeTypeEmailProvider.java

```diff
@@ -4,6 +4,13 @@
 
 public interface MimeTypeEmailProvider {
 
+    /**
+     * Get the sender email address without any further formatting. So for example, if the provider formats the sender
+     * email as <code>"Study Name" &lt;email@email.com&gtl</code>, This method should return only
+     * <code>email@email.com</code>.
+     */
+    String getPlainSenderEmail();
```

Comment:
It looks like the sender email will always be study.getSupportEmail(). (Which it probably should be, since verifying emails in SES seems to be a huge bottleneck, from a herding cats point of view.) Given this, how much of a refactor would it be so MimeTypeEmailProvider is an abstract class instead of an interface, and always knows to get the sender email from study.getSupportEmail()?

Similarly, MimeTypeEmail.getSenderAddress() seems to be the same in all providers, and the logic for generating and escaping them seems to be the same. Is it worth consolidating this to the provider, so we don't need to write new sender formatting code with each provider?